country_by_country/processor.py

```diff
@@ -34,17 +34,17 @@ def __init__(self, config: dict) -> None:
         # Report filter
         self.page_filter = pagefilter.from_config(config["pagefilter"])
 
-        # Table extraction from images
-        self.img_table_extractors = []
+        self.table_extractors = []
         self.table_cleaners = []
 
+        # Tables extraction
         if "table_extraction" in config:
-            img_table_extractors = config["table_extraction"]["img"]
-            self.img_table_extractors = [
-                img_table_extraction.from_config(name) for name in img_table_extractors
+            table_extractors = config["table_extraction"]
+            self.table_extractors = [
+                table_extraction.from_config(name) for name in table_extractors
             ]
 
-            # Table cleaning & reformatting
+            # Tables cleaning & reformatting
```

Comment:
```suggestion
            # Table cleaning & reformatting
```

Previous version reads prefectly in English